merkle-tree/src/constants/poseidon.rs

```diff
@@ -0,0 +1,80 @@
+use super::ZeroBytes;
+
+pub const ZERO_BYTES: ZeroBytes = [
```

Comment:
Should we name the zero_bytes according to height?
Or we could just create zero bytes for height 32 and just use the zero bytes we need for the specific height.
(The zero bytes of a lower height are always a subset of the zero bytes for a higher height.)

Reply:
Done.